### PR TITLE
Config with zope product

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,95 +1,103 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/zope-product
 [tox]
+minversion = 3.18
 envlist =
-    lint,
-    py27,
-    py27-zserver,
```
Comment (dataflake) [inline tox.ini:5]: This is getting rid of tests with ZServer. Intention or oversight?

**Review by dataflake — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (icemac) [reply tox.ini:5]: Sorry, this is an oversight, I am going to fix it now.

Comment (icemac) [reply tox.ini:5]: Fixed in https://github.com/zopefoundation/Products.PluggableAuthService/pull/86/commits/09237f2dc49a3f4e96142881b5404c94d1fffa28

**Review by dataflake — APPROVED** [review]

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**